.storybook/preview.tsx

```diff
@@ -74,6 +74,10 @@ const preview: Preview = {
             linkTo("Pages/Public/Breach listing")();
           }
 
+          if (path === "/terms/expiration-offer") {
```

Comment:
It doesn't look like this actually works anymore with the latest Storybook (I can't find the docs on it anymore either), but figured it couldn't harm in case they add it back or something.